### ISSUE TITLE
[FLAKE] List operators installed in the namespace

### ISSUE THREAD

**Issue by amitkrout** [openshift/odo#2924]
/kind bug

## What versions of software are you using?

**Operating System:**
supported
**Output of `odo version`:**
master
## How did you run odo exactly?
https://github.com/openshift/odo/blob/master/tests/integration/operatorhub/cmd_service_test.go#L26

**Comment by amitkrout** [comment]
Unable to reproduce it locally. Team, please paste the failure log if you find the same failure in CI

**Comment by kadel** [comment]
This is probably one instance of this https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_odo/2885/pull-ci-openshift-odo-master-v4.3-integration-e2e-benchmark/2105

**Comment by mik-dass** [comment]
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_odo/2885/pull-ci-openshift-odo-master-v4.3-integration-e2e-benchmark/2105#1:build-log.txt%3A443 @amitkrout is this related?

**Comment by mik-dass** [comment]
```

• Failure [0.809 seconds]
odo service command tests for OperatorHub
/go/src/github.com/openshift/odo/tests/integration/operatorhub/cmd_service_test.go:17
  When creating an operator backed service
  /go/src/github.com/openshift/odo/tests/integration/operatorhub/cmd_service_test.go:34
    should be able to create EtcdCluster from its alm example [It]
skipped 14 lines unfold_more
[odo] Operators available in the cluster
[odo] NAME                          CRDs
[odo] mongodb-enterprise.v1.4.5     MongoDB, MongoDBUser, MongoDBOpsManager
```
/retest

**Comment by amitkrout** [comment]
> https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_odo/2885/pull-ci-openshift-odo-master-v4.3-integration-e2e-benchmark/2105#1:build-log.txt%3A443 @amitkrout is this related?

It is not the same issue. I checked the report https://gist.github.com/kadel/2570427b9113ea0bb546f45723b7a643 where the same failure has been reported only once. Then this will be the second hit, i mean failure count 2.

<img width="1002" alt="Screen Shot 2020-04-20 at 5 46 04 PM" src="https://user-images.githubusercontent.com/19851115/79750767-23782780-832f-11ea-98c8-15d6de5d3e72.png">

I am will create a separate issue to track it.

**Comment by amitkrout** [comment]
> > https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_odo/2885/pull-ci-openshift-odo-master-v4.3-integration-e2e-benchmark/2105#1:build-log.txt%3A443 @amitkrout is this related?
> 
> It is not the same issue. I checked the report https://gist.github.com/kadel/2570427b9113ea0bb546f45723b7a643 where the same failure has been reported only once. Then this will be the second hit, i mean failure count 2.
> 
> <img alt="Screen Shot 2020-04-20 at 5 46 04 PM" width="1002" src="https://user-images.githubusercontent.com/19851115/79750767-23782780-832f-11ea-98c8-15d6de5d3e72.png">
> 
> I am will create a separate issue to track it.

Issue created -https://github.com/openshift/odo/issues/2943

**Comment by dharmit** [comment]
I've assigned that issue to me but how is it different from this one? They both are talking about the same thing.

**Comment by amitkrout** [comment]
> I've assigned that issue to me but how is it different from this one? They both are talking about the same thing.

May be you are right. This issue has been created based on the report https://gist.github.com/kadel/2570427b9113ea0bb546f45723b7a643 where we had atleast the flake info without logs. Moreover i do not see any other reason for failure except the reason i created #2943. So i am making it as a duplicate of #2943

**Comment by amitkrout** [comment]
duplicate of #2943
